recipe_scrapers/farmhousedelivery.py

```diff
@@ -0,0 +1,94 @@
+            for ingredients_marker_sibling in ingredients_marker_siblings:
+                if (
+                    isinstance(ingredients_marker_sibling, Tag)
+                    and ingredients_marker_sibling.name == "ul"
+                ):
+                    ingredients = ingredients_marker_sibling.findAll("li")
+                    return [
+                        normalize_string(ingredient.get_text(strip=True))
+                        for ingredient in ingredients
+                    ]
+
+        # Style 2
+        ingredients_marker = self.soup.find("p", text=re.compile(r"Ingredients"))
+        if ingredients_marker is not None:
+            ingredients = []
+            ingredients_marker_siblings = ingredients_marker.next_siblings
+            for ingredients_marker_sibling in ingredients_marker_siblings:
+                if (
+                    isinstance(ingredients_marker_sibling, Tag)
+                    and ingredients_marker_sibling.name == "p"
+                ):
+                    if ingredients_marker_sibling.get_text() == "Instructions":
+                        break
+                    else:
+                        ingredients.append(ingredients_marker_sibling)
+            return [
+                normalize_string(ingredient.get_text()) for ingredient in ingredients
+            ]
+
+        return None
+
+    def instructions(self):
+        # Style 1
+        instructions_marker = self.soup.find("p", text=re.compile(r"Instructions:"))
+        if instructions_marker is not None:
+            instructions_marker_siblings = instructions_marker.next_siblings
+            for instructions_marker_sibling in instructions_marker_siblings:
+                if (
+                    isinstance(instructions_marker_sibling, Tag)
+                    and instructions_marker_sibling.name == "p"
+                    and instructions_marker_sibling.get_text(strip=True) != ""
+                ):
+                    instructions = instructions_marker_sibling.findAll("span")
+                    return [
+                        normalize_string(instruction.get_text())
+                        for instruction in instructions
+                    ]
+
+        # Style 2
+        instructions_marker = self.soup.find("p", text=re.compile(r"Instructions"))
+        if instructions_marker is not None:
+            instructions = []
+            instructions_marker_siblings = instructions_marker.next_siblings
+            for instructions_marker_sibling in instructions_marker_siblings:
+                if (
+                    isinstance(instructions_marker_sibling, Tag)
+                    and instructions_marker_sibling.name == "p"
+                    and instructions_marker_sibling.get_text(strip=True) != ""
+                ):
+                    instructions.append(instructions_marker_sibling)
+            return [
+                normalize_string(instruction.get_text()) for instruction in instructions
+            ]
```

Comment:
Same as above extra loop comment.

Reply:
Thanks for the recommendation.  Code updated.